.travis.yml

```diff
@@ -16,15 +18,19 @@ git:
 #  - julia: nightly
 
 ## uncomment and modify the following lines to manually install system packages
-# addons:
-#   apt: # apt-get for linux
-#     packages:
-#     - ffmpeg
-#before_script: # homebrew for mac
+addons:
+   apt: # apt-get for linux
+     packages:
+     - libxt6
+     - libxrender1
+     - libxrender1
+     - libgl1-mesa-glx
+before_script: # homebrew for mac
+#  - sudo apt install libxt6 libxrender1 libgl1-mesa-glx #https://github.com/jheinen/GR.jl/issues/107
```

Comment:
Can we remove this completely?

Reply:
Yes, we can probably get rid of it. I added this because I had an error similar to this one:
#https://github.com/jheinen/GR.jl/issues/107

But I feel like it's the following line that fixed the error I was having on TRAVIS CI:
```julia
ENV["GKSwstype"] = "100"
```